index.js

```diff
@@ -106,7 +107,29 @@ module.exports = {
 		'use-isnan': 'warn',
 		'wrap-iife': ['error', 'inside'],
 
-		// react plugin
+		// jsx-a11y plugin https://github.com/evcohen/eslint-plugin-jsx-a11y
+		'jsx-a11y/accessible-emoji': 'warn',
+		'jsx-a11y/alt-text': 'warn',
+		'jsx-a11y/anchor-has-content': 'warn',
+		'jsx-a11y/anchor-is-valid': ['warn', {
+			aspects: ['noHref', 'invalidHref']
+		}],
+		'jsx-a11y/aria-activedescendant-has-tabindex': 'warn',
+		'jsx-a11y/aria-props': 'warn',
+		'jsx-a11y/aria-proptypes': 'warn',
+		'jsx-a11y/aria-role': ['warn', {ignoreNonDOM: true}],
+		'jsx-a11y/aria-unsupported-elements': 'warn',
+		'jsx-a11y/heading-has-content': 'warn',
+		'jsx-a11y/iframe-has-title': 'warn',
+		'jsx-a11y/img-redundant-alt': 'warn',
+		'jsx-a11y/no-access-key': 'warn',
+		'jsx-a11y/no-distracting-elements': 'warn',
+		'jsx-a11y/no-redundant-roles': 'warn',
+		'jsx-a11y/role-has-required-aria-props': 'warn',
+		'jsx-a11y/role-supports-aria-props': 'warn',
+		'jsx-a11y/scope': 'warn',
```

Comment:
I feel like a11y is a good practice but I'm not sure it belongs in the non-strict version.

Reply:
These are the bare basics, copied from CRA.  For ideal compatibility with CRA and potentially offloading future components with their stack, best to adopt now. Plus they're mostly common-sense warnings, no errors; functionality/feature based.

Reply:
Okie dokie.